examples/local-mode-tests/tests/module.test.ts

```diff
@@ -21,9 +21,23 @@ const isWindows = process.platform === "win32";
 let wranglerProcess: ChildProcess;
 
 beforeAll(async () => {
-  wranglerProcess = spawn("npx", ["wrangler", "dev", "--local"], {
-    shell: isWindows,
-  });
+  wranglerProcess = spawn(
+    "npx",
+    [
+      "wrangler",
+      "dev",
+      "src/module.ts",
+      "--local",
+      "--config",
+      "src/wrangler.module.toml",
+      "--port",
+      "9001",
```

Comment:
OOC why change the port?

Reply:
Ah, because this and the SW tests are run in parallel?

Reply:
yup correct. I'm writing notes for all these hacks because we'll have to solve them when we make this a first class api.